<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection setup</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection setup</string>
   </property>
   <widget class="QLabel" name="avatarLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>111</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Avatar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="avatarPixmap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>137</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeAvatarButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>217</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change avatar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectLANButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>109</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect LAN ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearHistoryButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>138</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear history</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordPasswordButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>44</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>18</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="hostLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>70</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>18</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
    <property name="buddy">
     <cstring>nameLineEdit</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="hostComboBox">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>44</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Fill to sign up...</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>44</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="avatarList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>267</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>267</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reconnectionCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconnection</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>avatarList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
